src/observability/summary.go

```diff
@@ -12,11 +12,11 @@ func GetSummaryData(request *opb.Request) (*opb.Response, error) {
 	resp := opb.Response{}
 	var err error = nil
 
-	if strings.Contains(request.Type, "process") || strings.Contains(request.Type, "file") || strings.Contains(request.Type, "network") {
+	if strings.Contains(request.Type, "process") || strings.Contains(request.Type, "file") || strings.Contains(request.Type, "network") || strings.Contains(request.Type, "syscall") {
```

Comment:
can we have a switch case instead of the if conditions? I think a switch will be more appropriate since we are targeting only the `request.Type` in both conditions. @seswarrajan any issues if we use a switch case?

Something like

```
switch request.Type {
	case "process","file","network","syscall":
        // handle operation
        case "ingress","egress":
        // handle ingress and egress
        default:
        // log a debug message saying no desired request.Types were found
}
```

Reply:
Since this need more discussion on code refactoring so this can be handled separately with code refactoring PR.